CHANGES.rst

```diff
@@ -5961,7 +5961,7 @@ cube_build
 datamodels
 ----------
 
-- Skip serializing `None` in datamodels to be compatible with `asdf>=2.8` [#5371]
+- Skip serializing `None` in datamodels to be compatible with ``asdf>=2.8`` [#5371]
```

Comment:
Just for my own understanding, why does `None` get single backquotes?  Or more generally, how do you decide how many backquotes to give certain things?